controller.go

```diff
@@ -58,7 +58,7 @@ import (
 	"github.com/docker/libnetwork/driverapi"
 	"github.com/docker/libnetwork/hostdiscovery"
 	"github.com/docker/libnetwork/netlabel"
-	"github.com/docker/libnetwork/sandbox"
+	"github.com/docker/libnetwork/platform"
```

Comment:
But platform means it's going to have several platform related interfaces. Nevertheless platform will confuse people to think something like a Docker platform. I know you meant it as a low level platform. In that case I would suggest something like `osal`(OS abstraction layer) and `osshim` etc.